external-crates/move/crates/move-analyzer/trace-adapter/src/trace_utils.ts

```diff
@@ -506,7 +516,9 @@ function processJSONLocalLocation(
     } else if ('Indexed' in traceLocation) {
         return processJSONLocalLocation(traceLocation.Indexed[0], localLifetimeEnds);
     } else {
-        throw new Error(`Unsupported location type: ${JSON.stringify(traceLocation)}`);
+        // there is nothing that needs to be done for 'Global' locations,
+        // neither with respect to lifetime nor with respect to location itself
```

Comment:
Just want to note that this is not exactly true, but for all intents and purposes this is fine. If we ever have a native function that takes a mutable reference and returns a field of that struct by mutable reference something would need to be done to support that here. However, I don't such a function will ever exist.

Reply:
I did have that in mind when writing this code, but I should still add a comment there along these lines for posterity